self-core-impl/src/test/java/com/selfxdsd/core/managers/StoredProjectManagerTestCase.java

```diff
@@ -202,61 +164,28 @@ public void pmUserReturnsProjects() {
         final Storage storage = Mockito.mock(Storage.class);
         Mockito.when(storage.projects()).thenReturn(all);
 
-
-        ProjectManager ProjectManager = new StoredProjectManager(1,
-                "123",
-                "zoeself",
-                Provider.Names.GITLAB,
-                "123token",
-                storage);
-
-        User PmUser = new StoredProjectManager(ProjectManager.id(),
-                ProjectManager.userId(),
-                ProjectManager.username(),
-                Provider.Names.GITLAB,
-                "123token",
-                storage).new PmUser(ProjectManager);
-
+        final ProjectManager manager = Mockito.mock(ProjectManager.class);
+        final User pmUser = new StoredProjectManager.PmUser(manager);
+        Mockito.when(pmUser.projects()).thenReturn(assigned);
         MatcherAssert.assertThat(
-                PmUser.projects(),
+                pmUser.projects(),
                 Matchers.is(assigned)
```

Comment:
@HoreaPetrila This test is still more complex than it should be. Just mock the manager to return a mock ``Projects``. You don't need to mock the whole Storage and everything :D